packages/cli/test/integration/auth.api.test.ts

```diff
@@ -84,6 +86,26 @@ describe('POST /login', () => {
 		const authToken = utils.getAuthToken(response);
 		expect(authToken).toBeDefined();
 	});
+
+	test('should throw AuthError for non-owner if not within users limit quota', async () => {
```

Comment:
Reminder to update here if we decide that `AuthError` (401) is not fitting for the users quota guard, see reasoning above.